docs/reference/docs/reindex.asciidoc

```diff
@@ -838,17 +706,9 @@ GET /_tasks/r1A2WoRbTwKZ516z6NEs5A:36619
 --------------------------------------------------
 // TEST[catch:missing]
 
-The advantage of this API is that it integrates with `wait_for_completion=false`
-to transparently return the status of completed tasks. If the task is completed
-and `wait_for_completion=false` was set, it will return a
-`results` or an `error` field. The cost of this feature is the document that
-`wait_for_completion=false` creates at `.tasks/task/${taskId}`. It is up to
-you to delete that document.
-
-
 [float]
 [[docs-reindex-cancel-task-api]]
-==== Works with the Cancel Task API
+===== Canceling reindex tasks
```

Comment:
This section technically covers a separate endpoint and should be documented separately.

Reply:
Just referencing the task API makes sense--and helps reduce the size of this long ref page.